utilities/xmlrpcpp/test/TestValues.cpp

```diff
@@ -27,8 +27,9 @@
 #include <stdlib.h>
 #include <string>
 
-#include "xmlrpcpp/XmlRpcValue.h"
```

Comment:
Nitpick: please avoid reordering existing code to keep the diff minimal.

Reply:
Addressed with 6f8500dcb139f13c50060bc6e44a4f748799a5a1